start_esp/nginx-auto.conf.template

```diff
@@ -48,10 +48,7 @@ http {
 % endif
 % endfor
 
-  set_real_ip_from  0.0.0.0/0;
-  set_real_ip_from  0::/0;
-  real_ip_header    X-Forwarded-For;
-  real_ip_recursive on;
```

Comment:
If we implement this feature in esp, are we going to turn on this override by default or create a option to turn it on?